openassessment/xblock/static/sass/oa/views/_oa-course-items-listing.scss

```diff
@@ -64,22 +64,23 @@
   }
 
   .ora-summary-value {
-    font-size: 2.2em;
+    font-size: 1.5em;
     padding-top: 3px;
+    text-overflow: ellipsis;
+    overflow: hidden;
```

Comment:
If this text can be hidden, I think that we should make sure that the `title` HTML attribute should be set on this element. That way, the user can hover over the value that has an ellipsis and get the full value in a tooltip.